projects/demo/src/app/datepicker/datepicker-date-input-explicit-wrapper.html

```diff
@@ -8,7 +8,7 @@
 <div clrForm>
   <h6>With [clrDate]</h6>
   <p></p>
-  <clr-date-container>
+  <clr-date-container [dateFormat]="'yyyy/MM'">
```

Comment:
Is this testing something specific here? We'd better keep format overrides in their own page.